src/Rules/Methods/ClosureCallParametersRule.php

```diff
@@ -0,0 +1,78 @@
+<?php declare(strict_types = 1);
+
+namespace PHPStan\Rules\Methods;
+
+use PhpParser\Node;
+use PHPStan\Analyser\Scope;
+use PHPStan\Reflection\ParametersAcceptorSelector;
+use PHPStan\Rules\FunctionCallParametersCheck;
+use PHPStan\Type\ClosureType;
+use PHPStan\Type\VerbosityLevel;
+
+class ClosureCallParametersRule implements \PHPStan\Rules\Rule
+{
+
+	/** @var \PHPStan\Rules\FunctionCallParametersCheck */
+	private $check;
+
+	public function __construct(FunctionCallParametersCheck $check)
+	{
+		$this->check = $check;
+	}
+
+	public function getNodeType(): string
+	{
+		return Node\Expr\MethodCall::class;
+	}
+
+	/**
+	 * @param \PhpParser\Node\Expr\MethodCall $node
+	 * @param \PHPStan\Analyser\Scope $scope
+	 * @return string[]
+	 */
+	public function processNode(Node $node, Scope $scope): array
+	{
+		$varType = $scope->getType($node->var);
+		if (!($varType instanceof ClosureType)) {
+			return [];
+		}
+
+		if (!($node->name instanceof Node\Identifier)) {
+			return [];
+		}
+
+		if ($node->name->name !== 'call') {
```

Comment:
I think we might be able to implement this not by introducing a new rule, but by changing the signature of the call() method on the fly. It would then be caught by CallMethodsRule automatically. ClosureType could override getMethod() implementation so that here the right method signature is used: https://github.com/phpstan/phpstan/blob/master/src/Rules/Methods/CallMethodsRule.php#L127